src/components/roomStack/StackRoom.tsx

```diff
@@ -147,10 +146,11 @@ const StackRoom: FunctionComponent<StackRoomProps> = ({ areaBox }) => {
   if (flippedRoom) {
     const { name, doorDirections } = flippedRoom;
     return (
-      <Group>
-        <Room box={roomBox} doorDirections={doorDirections} />
-        <RoomName box={roomBox} name={name} />
-      </Group>
+      <FlippedStackRoom
+        areaBox={areaBox}
+        name={name}
+        doorDirections={doorDirections}
+      />
```

Comment:
This component existed, I just realized I extracted it but forgot to start using it :sweat_smile: